src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -1,71 +1,78 @@
+import { BaseSyntheticEvent } from 'react';
+import { stardust } from '@nebula.js/stardust';
 import * as handleAccessibility from '../handle-accessibility';
 
 describe('handle-accessibility', () => {
-  let cell;
-  let rootElement;
-  let focusedCellCoord;
-  let setFocusedCellCoord;
-  let keyboard;
+  let focusType: string;
```

Comment:
this is only used for updateFocus, so you can move it back to inside that describe